gears/src/rest/handlers.rs

```diff
@@ -51,6 +52,50 @@ pub async fn node_info(
     Ok(Json(node_info))
 }
 
+pub async fn validatorsets_latest(
```

Comment:
Should this `fn` be part of gears?

Reply:
Actually not. It's the call to `tendermint` service and it should be only an endpoint that connects app with tendermint. The scheme with `tendermint` looks like:

tendermint asks app for validators to create validatorset -> frontend asks app to ask tendermint for validatorset

Btw, each app with staking module may ask for validators by `validators` query.